synapse/models/economic.py

```diff
@@ -296,5 +309,26 @@ def getModelDefs(self):
                     ('price:high', ('econ:price', {}), {
                         'doc': 'The high price of the security'}),
                 )),
+
+                ('econ:bank:aba:rtn', {}, ()),
+                ('econ:bank:swift:bic', {}, ()),
+
+                ('econ:bank:account', {}, (
+
+                    ('number', ('str', {'regex': '[0-9]+'}), {
+                        'doc': 'The account number.'}),
+
+                    ('aba:rtn', ('econ:bank:aba:rtn', {}), {
+                        'doc': 'The ABA routing transit number for the bank which issued the account.'}),
```

Comment:
Should this be "account number" to match the later descriptions of "bank which issued the account number"?

Reply:
We may or may not have the actual account number, so i'd prefer to keep the slightly more inclusive docs.